aea/manager/manager.py

```diff
@@ -516,12 +516,8 @@ def start_agent(self, agent_name: str) -> "MultiAgentManager":
         if self._is_agent_running(agent_name):
             raise ValueError(f"{agent_name} is already started!")
 
-        agent_alias.issue_certificates()
         aea = agent_alias.get_aea_instance()
-        # override build dir to project's one
-        aea.DEFAULT_BUILD_DIR_NAME = os.path.join(
-            agent_alias.project.path, aea.DEFAULT_BUILD_DIR_NAME
-        )
```

Comment:
The order was wrong, and also should be done as part of getting the instance, so moved it.